login-workflow/package.json

```diff
@@ -36,7 +39,7 @@
         "@mui/icons-material": "^5.10.15",
         "@mui/material": "^5.10.15",
         "date-fns": "^2.14.0",
-        "i18next": "^19.3.4 || ^20.0.0",
+        "i18next": "^22.0.0",
```

Comment:
Is 22 required or does it still work with 19 and 20? (i.e, should this be a dependency change or a devDependency change?)

Reply:
I can try to downgrade, but I think there was method name updates between the versions that are breaking